internal/testutil/acc/name.go

```diff
@@ -36,3 +40,17 @@ func RandomIP(a, b, c byte) string {
 func RandomEmail() string {
 	return fmt.Sprintf("%s-%s@mongodb.com", prefixName, acctest.RandString(10))
 }
+
+func ProjectIDGlobal(tb testing.TB) string {
+	tb.Helper()
+	return projectID(tb, prefixProjectKeep+"-global")
+}
+
+func projectID(tb testing.TB, name string) string {
+	tb.Helper()
+	SkipInUnitTest(tb)
```

Comment:
curious: why we do this? should we also document it?

Reply:
i've added more documentation to SkipInUnitTest, let me know if it helps